### ISSUE TITLE
Cannot find source file warnings

### ISSUE THREAD

**Issue by kennethkoontz** [webpack-contrib/source-map-loader#18]
Not sure if this is the correct place to report this issue but when adding new node packages, sometimes I get errors as such:

``` bash
WARNING in ./~/ng2-modal/index.js
Cannot find source file '../../../src/index.ts': Error: Cannot resolve 'file' or 'directory' ../../../src/index.ts in /Users/kenneth/Projects/benchmark/web-client/node_modules/ng2-modal
```

Adding the following `exclude` entry to my webpack configuration suppresses the warning:

``` js
      {
        test: /\.js$/,
        loader: 'source-map-loader',
        exclude: [
          helpers.root('node_modules/ng2-modal')
        ]
      }
```

What can I do to resolve this without excluding to suppress the warning?

**Comment by jacob-robertson** [comment]
I'm seeing this as well.  It's very inconvenient, since the warnings make the actual output difficult to read.
I've included my sample configuration and output.

[SampleConfig.zip](https://github.com/webpack-contrib/source-map-loader/files/740567/SampleConfig.zip)


[Sample output](https://i.imgur.com/34VMS78.png)

**Comment by johanssonfredrik** [comment]
I am also seeing this problem. 

We have a npm linked dependency that we are unable to exclude from the sourcemapping. It seems that the path resolves incorrectly. If i exclude relative path 'node_modules/<linked lib>' it do not exclude it but if i give a absolute path the exclusion works.

**Comment by michael-ciniawsky** [comment]
Eventually fixed by #68

**Comment by alecsgone** [comment]
Hi, @kennethkoontz will you be so kind and tell me what’s ‘helpers.root’ where that came from?
